### PR TITLE
[Application] Update README.md File @open sesame 07/20 10:38

### PR DESCRIPTION
**Pull request by DonghakPark**
- Update README res/mnist.ini part (Split Optimizer, LearningRateScheduler)
- Update align code comment

Signed-off-by : DongHak Park <donghak.park@samsung.com>

### CODE REVIEW

File: Applications/MNIST/README.md [hunk 1]
```diff
@@ -41,18 +41,25 @@ int getBatch_train(float **inVec, float **inLabel, bool *last,
...
-Epochs = 1500		        # Epochs
-Optimizer = adam 	        # Optimizer : adam (Adamtive Moment Estimation)
-Loss = cross  		        # Loss function : cross ( for cross entropy )
-Save_Path = "model.bin"  	# model path to save / read
-batch_size = 32		        # batch size
-beta1 = 0.9 		        # beta 1 for adam
-beta2 = 0.999	            # beta 2 for adam
-epsilon = 1e-7	            # epsilon for adam
+Type = NeuralNetwork	            # Network Type : Regression, KNN, NeuralNetwork
+Epochs = 1500		                # Epochs
```
Comment (songgot) [inline Applications/MNIST/README.md:48]: How about matching the indentation with other '#'s?
and LGTM

**Review by songgot — APPROVED** [review]
LGTM

File: Applications/MNIST/README.md (hunk 1 above)
Comment (DonghakPark) [reply Applications/MNIST/README.md:48]: That's not what I meant. what I saw was different because of the editor. 
Thank you.!!!

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**